src/LaravelSamlIdpServiceProvider.php

```diff
@@ -72,11 +72,6 @@ public function offerPublishing()
             $this->publishes([
                 __DIR__ . '/../config/samlidp.php' => config_path('samlidp.php'),
             ], 'samlidp_config');
-
-            // Create storage/samlidp directory
-            if (!file_exists(storage_path() . "/samlidp")) {
-                mkdir(storage_path() . "/samlidp", 0755, true);
-            }
```

Comment:
This directory is already created when running the `samlidp:cert` command, so it can safely be removed here.